src/main/java/com/puppycrawl/tools/checkstyle/ant/CheckstyleAntTask.java

```diff
@@ -627,12 +627,12 @@ public void setType(FormatterType type) {
         }
 
         /**
-         * Set the file to output to.
+         * Set the file target.
          *
-         * @param destination destination the file to output to
+         * @param target for file output
          */
-        public void setTofile(File destination) {
```

Comment:
this looks kind of a typo imho
`setTofile` ->
`setToFile` (CamelCase)

Reply:
I know this term commonly from [Path.toFile](https://docs.oracle.com/javase/8/docs/api/java/io/File.html#toPath--) as it sounds like a converter method and not like a field or property in first place.